src/dbSta/src/dbReadVerilog.cc

```diff
@@ -138,7 +137,8 @@ Cell* dbVerilogNetwork::findAnyCell(const char* name)
 
 void dbReadVerilog(const char* filename, dbVerilogNetwork* verilog_network)
 {
-  sta::readVerilogFile(filename, verilog_network);
+  sta::VerilogReader* verilog_reader = new sta::VerilogReader(verilog_network);
+  verilog_reader->read(filename);
```

Comment:
wont this leak memory?

Reply:
I'm trying to figure this out. If I delete the pointer, a bunch of tests fail with the error `[ERROR STA-1398] top is not a verilog module.`

Reply:
@gadfort my last commit fix the leak by mimic the old behavior. I had to make the verilog reader a class member, in order to avoid creating multiple readers, and I delete it once the link design is complete.